File: 2535-difference-between-element-sum-and-digit-sum-of-an-array/2535-difference-between-element-sum-and-digit-sum-of-an-array.java
class Solution {
    public int differenceOfSum(int[] nums) {
        int element = 0, digit = 0;
        for(int i=0;i<nums.length;i++){
            element+=nums[i];
            if(nums[i] < 10){
                digit+=nums[i];
            }else{
                while(nums[i]!=0){
                    int temp = nums[i]%10;
                    digit+=temp;
                    nums[i]/=10;
                }
            }
        }
        return Math.abs(element-digit);
    }
}

File: 2535-difference-between-element-sum-and-digit-sum-of-an-array/2535-difference-between-element-sum-and-digit-sum-of-an-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var differenceOfSum = function(nums) {
    let elementSum = 0, digitSum = 0;
    for(let i=0;i<nums.length;i++){
        elementSum+=nums[i];
        if(nums[i] >= 10){
            while(nums[i]!=0){
                let rem = nums[i]%10;
                digitSum+=rem;
                nums[i] = Math.floor(nums[i]/10);
            }
        }else{
            digitSum+=nums[i];
        }
    }
    return Math.abs(elementSum-digitSum);
};